### PR TITLE
Adding Prettier and formatting the code 

### PR DESCRIPTION
**Pull request by ravjot07**
contributes towards #876

### CODE REVIEW

**Comment by ravjot07** [comment]
@aaronreed708 sir can you take a look why my tests are failing and is it write approach from my side to user prettier for code formatting as i have seen it various open source organisations and in their CI/CD Pipeline. I would also recommend adding style lint with quality tools like Prettier and ESLint.

File: code/src/app.ts [hunk 1]
```diff
@@ -2,65 +2,70 @@
...
+  app.use(express.json());
+  app.use(cookieParser());
+  addAuthMiddleware(app, cfg);
+  app.use('/api/*', bodyParser.json());
+  registerThemesEndpoint(app);
+  app.use(handleError);
+  app.use(express.static(path.join(__dirname, '../src/ui/build')));
+  app.get('/*', function (req: Request, res: Response) {
+    res.sendFile(path.join(__dirname, '../src/ui/build', 'index.html'));
+  });
```
Comment (Sweetdevil144) [inline code/src/app.ts:58]: We've been using 2 tab spaces consistently thorough-out the Project. This leads to decreased code quality w.r.t previous practices.

**Comment by Sweetdevil144** [comment]
> I would also recommend adding style lint with quality tools like Prettier and ESLint.

ESlint is already included in another PR. Please review #856 for more details.

**Comment by Sweetdevil144** [comment]
I guess you'd cloned the `main` branch instead of Dev before. Main is not the latest branch and needs regular updating which are resolved after fixed time intervals from `dev` branch by maintainers. Branch Violations via this PR.

**Comment by Sweetdevil144** [comment]
I'd recommend closing this PR for good and refactoring changes in another.

File: code/src/app.ts (hunk 1 above)
Comment (Sweetdevil144) [reply code/src/app.ts:58]: Can you refine such things in the `prettier-config` too?

**Comment by aaronreed708** [comment]
@ravjot07 it would be great if you could base your changes on a branch that derives from the `dev` branch as mentioned by @Sweetdevil144.  I can see in your `testing` commit that you are not on the correct branch.  The conflicts at the end here would also need to be resolved prior to review.  Thank you.  Prettier can seems to make the code much more consistent, I appreciate your work on this!

File: code/src/app.ts (hunk 1 above)
Comment (aaronreed708) [reply code/src/app.ts:58]: If possible, I think a four space indentation would be best.  That is predominant in the code currently.

**Comment by ravjot07** [comment]
> @ravjot07 it would be great if you could base your changes on a branch that derives from the `dev` branch as mentioned by @Sweetdevil144. I can see in your `testing` commit that you are not on the correct branch. The conflicts at the end here would also need to be resolved prior to review. Thank you. Prettier can seems to make the code much more consistent, I appreciate your work on this!

yes sir i will make another pr regarding this

**Pull request closed without merging**